tv_apps/fling-player/js/fling_player.js

```diff
@@ -1,20 +1,78 @@
-/* global VideoPlayer, Connector */
+/* global VideoPlayer, Connector, SimpleKeyNavigation, KeyNavigationAdapter,
+          evt, mDBG
+ */
 (function(exports) {
   'use strict';
 
+  mDBG.test(() => { // TMP
+    window._TMP_duration = 600;
+    window._TMP_current = window._TMP_duration * 0.33;
+  });
```

Comment:
I'm not sure if we need to leave test codes inside our codebase. Can't we do this in unit test or integration test?